.packit.yml

```diff
@@ -6,6 +6,7 @@ actions:
   post-upstream-clone:
     - ./autogen.sh
     - ./configure
+    - sed -i 's/use_cockpit\ 1/use_cockpit 0/g' anaconda.spec
```

Comment:
Honestly I think we should have "make release-cockpit" to solve this better way.

Reply:
Hmm, can your elaborate? I am not sure I understand.

Reply:
discussion IRC